home/ivp-planning/advanced-tools/local-library.md

```diff
@@ -0,0 +1,33 @@
+# Local Library
+
+  
+{% hint style="info" %}  
+"iVP Library" will not show up if the folder is empty, and will show up as soon as items are added to it.  
+The folders name can be changed at any time.  
+Supported file-types include Images, .plan and 3D-Models.  
+{% endhint %}
+
+You can import your own files/objects from your PC into iVP by using the "Import" option.  
+
+![](.gitbook/assets/3d-import-file.png)
+
+When importing your file, you have the option to change its parameters, as well as the option to add it to your library.  
+
+![](.gitbook/assets/3d-import-addtolibrary.png)  
+
+Once you have added an item to your library, the folder "iVP Library" will appear at the bottom of the library window.
+From now on every item that you import, and chose to add to your library, can be found inside the "iVP Library" folder.  
+
+The Order of your items can only be changed inside the folder, **not** inside iVP-Planning.
+
+![](.gitbook/assets/3d-import-library.png)  
+
+Your "iVP Library" folder will be created in the default iVP-Directory.
+You can click on the "Open" button next to any object inside the fodler to open the directory directly.
```

Comment:
Same here, take a few more sentences to explain what you actually mean. People won't know what the "Open button" is. Something along the lines of:

When hovering an entry in the library, a button with a folder icon will appear. Clicking it will open the respective folder or select the respective file in your file browser.

would be way easier to understand in my opinion.